package.json

```diff
@@ -202,12 +202,13 @@
     "html-webpack-plugin": "4.5.0",
     "husky": "4.3.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.6.0",
-    "jest-circus": "^26.6.3",
-    "jest-junit": "12.0.0",
+    "jest": "28.1.0",
+    "jest-circus": "^28.1.0",
```

Comment:
Why are we installing jest-circus? 
<img width="851" alt="image" src="https://user-images.githubusercontent.com/4605522/172486641-8acced7b-3a6c-4b5a-8d7b-e064fbb4a16f.png">
https://www.npmjs.com/package/jest-circus